packages/@okta/vuepress-site/guides/custom-error-pages/use-macros/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Use Macros
+---
+The following macros contain the configuration parameters for certain page elements. These macros inject specific content or functionality automatically.
+
+The graphic highlights and numbers the macros in the html editor, and then displays the corresponding number on the error page for each macro and what text or image is affected.
+
+![Macros on Error Page](/img/custErrorPage.png "Macros on Error Page")
+
+#### <span v-pre>`{{orgName}}`</span> 
+
+This macro controls the org name title that appears on the error page's browser tab. You can make changes to this text by removing the macro and adding your own.
+
+For example:
+```javascript
+<title>Your Text Here - {{errorSummary}}</title>
+```
+
+#### <span v-pre>`{{errorSummary}}`</span>
+
+This macro controls the error title text that appears on the error page's browser tab. You can make changes to this text by removing the macro and adding your own.
+
+For example:
+```javascript
+<title>{{orgName}} - Your Text Here</title>
+```
+
+> Note: Removing this <span v-pre>`{{errorSummary}}`</span> macro doesn't affect the <span v-pre>`{{errorSummary}}`</span> macro (6) lower on the page.
+
+#### <span v-pre>`{{bgImageUrl}}`</span>
+
+This macro controls the background image of the entire error page. You can configure which background image this macro displays using the **Sign-In Configuration** option accessed by selecting **Customization**, and then **Appearance** from the Developer Console. 
+You can also change which background image appears by removing the macro in the HTML editor and adding your own background image URL. 
+
+For example:
+
+```html
+<div class="login-bg-image" style="background-image: url('https://example.com//YourBackgroundImage.png)"></div>
```

Comment:
looks like the url is missing an end quote 

`<div class="login-bg-image" style="background-image: url('https://example.com//YourBackgroundImage.png')"></div>`